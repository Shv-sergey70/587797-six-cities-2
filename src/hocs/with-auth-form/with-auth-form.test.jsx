import React from 'react';
import renderer from 'react-test-renderer';
import TestWrappedComponent from './test-component';

it(`HoC with-auth-form correctly renders`, () => {
  const tree = renderer.create(
      <TestWrappedComponent/>).toJSON();

  expect(tree).toMatchSnapshot();
});
