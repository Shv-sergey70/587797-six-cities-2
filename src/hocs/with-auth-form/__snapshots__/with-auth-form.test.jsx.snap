// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HoC with-auth-form correctly renders 1`] = `
<div>
  TEST
  <input
    name="testName"
    onChange={[Function]}
    value="testValue"
  />
</div>
`;
